### ISSUE TITLE
why still use deprecated object.observe api? such as, Collection(name, optionsopt)!

### ISSUE THREAD

**Issue by anlexN** [techfort/LokiJS#655]

**Comment by Viatorus** [comment]
We removed observe in LokiJS2.

**Comment by anlexN** [comment]
what time is LokiJS2 published?

**Comment by Viatorus** [comment]
Check out https://github.com/LokiJS-Forge/LokiJS2. Its still beta mostly only the documentation is missing.

**Comment by obeliskos** [comment]
I agree this should not be in lokijs2.   We left this in there to not break compatibility with any existing solutions which depended on it.  

Did node.js ever totally remove support for object.observe or is this only available via polyfill?  

@Cortys, not sure if you are familiar (as you likely moved on), but do you know if there are any viable platforms where this might still be used?

**Comment by Cortys** [comment]
@obeliskos As far as I know not. `Object.observe` was only ever supported by Chrome and Node.js and both have removed it in favor of ES6 Proxies in v50/v6. So only Node 4 users could potentially be affected by dropping the autoupdate implementation. Since Node 4's EOL is on April, 30th there probably is no reason to keep it.

**Comment by anlexN** [comment]
please let's remove observe, follow standards. it is a common view.
